src/apps/activity_assignments/domain/assignments.py

```diff
@@ -79,3 +80,15 @@ def validate_assignments(cls, values):
 
 class ActivitiesAssignmentsDelete(InternalModel):
     assignments: list[ActivityAssignmentDelete]
+
+
+class AssignmentsSubjectCounters(PublicModel):
```

Comment:
Nit: objects of this class don't get returned directly by the API, but are only used internally. Should it be `InternalModal`? I'm not totally clear on the difference…
```suggestion
class AssignmentsSubjectCounters(InternalModel):
```